cmd/broker/ingress/main.go

```diff
@@ -156,13 +209,31 @@ func (h *handler) serveHTTP(ctx context.Context, event cloudevents.Event, resp *
 		return nil
 	}
 
+	ctx, _ = tag.New(ctx, tag.Insert(TagBroker, h.brokerName))
```

Comment:
I'm not a fan of ignoring errors. Are we sure this will never error? If not, do we really want no behavior change on failure?

Reply:
Looks like an error is only returned if [the tag or value are invalid](https://github.com/census-instrumentation/opencensus-go/blob/5ae9166f90ff989c557e45fcb9592ac59afa0fa0/tag/map.go). Validity is [currently defined](https://github.com/census-instrumentation/opencensus-go/blob/5ae9166f90ff989c557e45fcb9592ac59afa0fa0/tag/validate.go) as printable standard ASCII (32-126) with length <= 255.

So in this specific case, because all tag keys and values are constants, no error will occur. But in other cases where non-constant values are accepted, an error could occur.

Behavior changes available are erroring the request or changing the key and/or value to be valid. Neither of these seem great to me - we don't want to error the request because this is a metrics error, not a user error. Trying to turn an invalid valid into a valid one sounds difficult and potentially costly.

If we do not want a behavior change, I believe the correct action is to ignore the error. If we wanted to be friendly, we could log it, but I'd need to trust [zap's log sampling implementation](https://github.com/uber-go/zap/blob/master/FAQ.md#why-sample-application-logs). These logs could easily cause terrible performance.

I'd be ok with logging this at Debug level, that way the developer or user must request the change and is hopefully aware of the potential for performance issues.